openxr/src/generated.rs

```diff
@@ -2365,6 +2794,12 @@ pub mod raw {
         pub const NAME: &'static [u8] = sys::EXT_HP_MIXED_REALITY_CONTROLLER_EXTENSION_NAME;
     }
     #[derive(Copy, Clone)]
+    pub struct PalmPoseEXT {}
+    impl PalmPoseEXT {
+        pub const VERSION: u32 = sys::EXT_palm_pose_SPEC_VERSION;
+        pub const NAME: &'static [u8] = sys::EXT_PALM_POSE_EXTENSION_NAME;
+    }
+    #[derive(Copy, Clone)]
     pub struct UuidEXT {}
```

Comment:
It looks like this isn't actually being generated correctly to begin with, unfortunately.

Reply:
so the extension and the actual UUID struct have the same name. the uuid struct is generated [here](https://github.com/kcking/openxrs/blob/a385cf4c2329a002e95d943b41e6dcdd2b49f2c2/sys/src/generated.rs#L6849) which looks right to me

Reply:
NVM, this is the extension struct, lol.